### ISSUE TITLE
An error has occurred. Ref: 120031

### ISSUE THREAD

**Issue by sverunna** [sverunna/zscaler-docker#1]
An error is thrown in certain conditions. Needs investigation

`An error has occurred. Ref: 120031`

**Comment by Michael-Ricart** [comment]
same here, some news about it ?

**Comment by sverunna** [comment]
@Michael-Ricart what are you trying to do? Which OS are you using?

**Comment by mattmiller87** [comment]
MacOS `15.3.1 (24D70)`. trying with `--platform linux/amd64`

**Comment by onehalf3570** [comment]
Yeah, this error occurs during initial 'docker run'. What is printing it, the docker itself?

**Comment by onehalf3570** [comment]
Any plans to release the source code? At the moment it's hard to debug and is also hard to do any audit (what is 'zscaler' binary that is run in the container from the Dockerfile?)
